src/index.html

```diff
@@ -9,6 +9,65 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <div class="logo">
+        <a href="#home">
+          <img
+            class="logo__img"
+            src="images/logo.png"
+            alt="Moyo logo"
+          >
+        </a>
+
+      </div>
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a class="nav__link is-active" href="#apple">
+              apple
+            </a>
+          </li>
+          <li class="nav__item">
+            <a class="nav__link" href="#samsung">
+              samsung
+            </a>
+          </li>
+          <li class="nav__item">
+            <a class="nav__link" href="#smartphones">
+              smartphones
+            </a>
+          </li>
+          <li class="nav__item">
+            <a
+              class="nav__link"
+              data-qa="hover"
+              href="#computers"
+            >
+              laptops & computers
+            </a>
+          </li>
+          <li class="nav__item">
+            <a class="nav__link" href="#gadgets">
+              gadgets
+            </a>
+          </li>
```

Comment:
as those are on the same level of nesting and there are quite a lot of them - you can add some empty lines between them for better readability
```suggestion
          <li class="nav__item">
            <a class="nav__link is-active" href="#apple">
              apple
            </a>
          </li>
          
          <li class="nav__item">
            <a class="nav__link" href="#samsung">
              samsung
            </a>
          </li>
          
          <li class="nav__item">
            <a class="nav__link" href="#smartphones">
              smartphones
            </a>
          </li>
          
          <li class="nav__item">
            <a
              class="nav__link"
              data-qa="hover"
              href="#computers"
            >
              laptops & computers
            </a>
          </li>
          
          <li class="nav__item">
            <a class="nav__link" href="#gadgets">
              gadgets
            </a>
          </li>
```